supplier_app/tests/test_view.py

```diff
@@ -520,155 +526,97 @@ def setUp(self):
             'address_form-state': 'Mendoza',
             'address_form-country': 'Argentina',
         }
-        self.kwargs = {
+        self.kwargs_taxpayer = {
             'taxpayer_id': self.taxpayer.id,
         }
+        self.kwargs_address = {
+            'address_id': self.address.id
+        }
         self.taxpayer_id = self.taxpayer.id
 
-    def test_get_success_url_should_redirect_to_details_view_when_click_in_update_button(self):
+    def test_get_detail_view(self):
```

Comment:
Please provide context of what are you testing on this specific test on the name, or add a comment inside

Reply:
resolved